.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,90 @@
+name: Build docs
+
+on:
+  pull_request:
+    branches:
+      - zpedro/docs_monorepo
```

Comment:
Noting this

Reply:
oops, thanks!

Reply:
added a `paths` directive so it also builds if someone makes changes to the docs and forgets to label the PR, or accidentally pushes changes to docs